severity.go

```diff
@@ -0,0 +1,6 @@
+package gofakeit
+
+// Severity will generate a random severity level
+func Severity() string {
+	return getRandValue([]string{"severity", "level"})
```

Comment:
I'm thinking if there should be two methods, one for short and one for the complete log level name. @brianvoe what do you think?

Reply:
Agreed